test/e2e/data/cni/antrea.yaml

```diff
@@ -0,0 +1,480 @@
+---
+# source: https://github.com/vmware-tanzu/antrea/releases/download/v0.1.0/antrea.yml
```

Comment:
using this version definitely does not seem right :)
This is the first version of Antrea (from 2019), and the latest version is v0.13.1

is it possible to check-in a version number in a file, and download the YAML automatically? Or does the whole YAML need to be checked-in and updated each time?

Reply:
I think the thing responsible for applying CNI is prepared to receive yaml that can be applied to the cluster.
I've updated to use the v0.13.1 yaml